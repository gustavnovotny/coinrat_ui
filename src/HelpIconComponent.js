import React from "react"
import {Icon, Intent} from "@blueprintjs/core"
import {Tooltip2} from "@blueprintjs/labs"

type Props = {
  helpText: string
}

const HelpIconComponent = ({helpText}: Props) => {
  return <Tooltip2 content={helpText}>
    <Icon iconSize={Icon.SIZE_STANDARD} intent={Intent.PRIMARY} iconName="pt-icon-info-sign"/>
  </Tooltip2>
}

export default HelpIconComponent
