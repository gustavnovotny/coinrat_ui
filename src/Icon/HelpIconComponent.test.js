// @flow
import React from 'react'
import renderer from 'react-test-renderer'
import HelpIconComponent from "./HelpIconComponent"

it('renders correctly', () => {
  const tree = renderer.create(<HelpIconComponent helpText="Trololoooooo"/>).toJSON()
  expect(tree).toMatchSnapshot()
})
