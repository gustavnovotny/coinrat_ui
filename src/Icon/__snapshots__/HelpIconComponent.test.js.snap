// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<span>
  <div
    className="pt-popover-target"
    onBlur={[Function]}
    onFocus={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <span
      className="pt-icon-standard pt-icon-info-sign pt-intent-primary"
      disabled={undefined}
      tabIndex={0}
    />
  </div>
</span>
`;
